ai/ai-starter-kit/helm-chart/ai-starter-kit/files/ray.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e4a6ac7c-5c73-42a9-8b74-420788321543",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install ray==2.41.0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "788f1517-251c-4171-af7d-f4c7a5073d71",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install numpy mlflow tensorflow \"ray[serve,default,client]\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8deec5c-6239-4087-8a4d-27c091e9fc3c",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
```

Comment:
Can this cell work without a GPU? I got error `failed call to cuInit: INTERNAL: CUDA error: Failed call to cuInit: UNKNOWN ERROR (303)` after running it.